components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -75,6 +76,9 @@ @implementation MDCMultilineTextField
 - (instancetype)initWithFrame:(CGRect)frame {
   self = [super initWithFrame:frame];
   if (self) {
+    _textView = [[MDCIntrinsicHeightTextView alloc] initWithFrame:CGRectZero];
+    [self setupTextView];
```

Comment:
[self setupTextView] is in both init* functions.  Move to commonInit?

Reply:
This one cannot be moved. It is a dependency for the fundament which is a dependency for the commonInit. I will document that.

Reply:
SGTM.